### PR TITLE
doc: Add LoRa PHY packet format

### PR DESCRIPTION
**Pull request by pradeeka7**
#### Summary

This page explains the PHY structure of the LoRa packet.

#### Screenshots

image

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [X] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/lorawan/lora-phy-format/_index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "LoRa Physical Layer Packet Format"
+section: Additional Information
+description: ""
+weight:
+---
+
+LoRa uses two types of packet formats for data transmission: explicit and implicit. 
+
+In **explicit** mode, a LoRa packet includes the following elements.
```
Comment (nejraselimovic) [inline doc/content/lorawan/lora-phy-format/_index.md]:
```suggestion
In **explicit** mode, a LoRa packet includes the following elements:
```

File: doc/content/lorawan/lora-phy-format/_index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+LoRa uses two types of packet formats for data transmission: explicit and implicit. 
+
+In **explicit** mode, a LoRa packet includes the following elements. 
+
+- **Preamble** is used to synchronize the receiver with the transmitter. It MUST consist of 8 symbols for all regions as mentioned in the LoRaWAN Regional Parameters document. However, the radio transmitter will add another 4.25 symbols resulting in a final preamble length of 8 + 4.25 = 12.25 symbols.
+
+- **PHDR (Physical Header)** is an optional element only present in the explicit mode that contains information about,
+Payload size - indicates the size of the payload.
+Cyclic Redundancy Check (CRC) - indicates the presence or absence of the Payload CRC.
```
Comment (nejraselimovic) [inline doc/content/lorawan/lora-phy-format/_index.md]:
```suggestion
- **PHDR (Physical Header)** is an optional element only present in the explicit mode that contains information about payload size and CRC (Cyclic Redundancy Check).
```

Comment (nejraselimovic) [reply doc/content/lorawan/lora-phy-format/_index.md]: If you use PHDR (Physical Header), please be consistent and use CRC (Cyclic Redundancy Check) and not Cyclic Redundancy Check (CRC). This improves readability.

File: doc/content/lorawan/lora-phy-format/_index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+- **PHDR_CRC (Header CRC)** is an optional field that contains an error detecting code for correcting errors in header.
+
+The PHDR and PHDR_CRC are encoded with the Coding Rate of 4/8.
+
+- **PHYPayload** contains the complete frame generated by the MAC layer. The maximum payload size varies by Data Rate (DR) and is region-specific.
```
Comment (nejraselimovic) [inline doc/content/lorawan/lora-phy-format/_index.md]: DR (Data Rate)

File: doc/content/lorawan/lora-phy-format/_index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+- **CRC** is an optional field that contains an error detecting code for correcting errors in the payload of uplink messages.
+
+The PHYPayload and CRC are encoded with one of the Coding Rates (4/5, 4/6, 4/7, or 4/8). The complete frame is then sent using one of the Spreading Factors used in the LoRaWAN (SF = 7 to 12).
```
Comment (nejraselimovic) [inline doc/content/lorawan/lora-phy-format/_index.md]:
```suggestion
The PHYPayload and CRC are encoded with one of the Coding Rates (4/5, 4/6, 4/7, or 4/8). The complete frame is then sent using one of the Spreading Factors (SF = 7 to 12).
```

**Review by nejraselimovic — APPROVED** [review]
@pradeeka7 this will be good to go after you make these little changes I requested. Please tag me when you're done.

**Comment by pradeeka7** [comment]
@nejraselimovic fixed as per your suggestions.

**Pull request merged**